deploy/helm/kubecf/values.yaml

```diff
@@ -483,11 +483,12 @@ eirini:
       enable: false
 
 bits:
+  download_eirinifs: false
   global:
     labels: {}
     annotations: {}
     images:
-      bits_service: eirini/bits-service:2.36.0@sha256:4cf84e13890890f5d8443a5e6e129b701d524f51d35c9c4295a0562ed8bb1bb2
+      bits_service: registry.suse.com/cap-staging/bits-service:bits-1.0.15-15.1.6.2.220-24.2
```

Comment:
We probably want a non-staging image eventually right? (But then that's probably more "production" than the old value anyway)

Reply:
The `download_eirinifs` flag controls the `bits` service init container `download-eirini-rootfs`. False disables that container (exclusion from template result).